repository-data/development/src/main/resources/hcm-content/content/documents/lks/home.yaml

```diff
@@ -4,17 +4,19 @@
   jcr:mixinTypes: ['hippo:named', 'hippo:versionInfo', 'mix:referenceable']
```

Comment:
Think we need to manually translate `Hero` `Header` & `Text` into `Page title` & `summary` on all the Home page documents during release.